<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCreateRsaKeyBase</class>
 <widget class="QDialog" name="DialogCreateRsaKeyBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelUsername">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelIP">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPort">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPassword">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditIP">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPort">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditUsername">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPassword">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonExit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateRsaKeyButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>240</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Create RSA key</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
